BootloaderCorePkg/Library/MpInitLib/Ia32/MpInitLib.c

```diff
@@ -30,17 +35,26 @@ UINT32                             mMpInitPhase = EnumMpInitNull;
 VOID
 SmmRebase (
   IN UINT32  Index,
-  IN UINT32  ApicId
+  IN UINT32  ApicId,
```

Comment:
Do you mind adding the function comment block ?

Reply:
Added.